plugins/modules/mso_backup_schedule.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: mso_backup_schedule
+short_description: Manages backup schedules
+description:
+- Manage backup schedules on Cisco ACI Multi-Site.
+author:
+- Akini Ross (@akinross)
+options:
+  start_date:
+    description:
+    - The date to start the scheduler in format YYYY-MM-DD
+    - If no date is provided, the current date will be used.
+    type: str
+  start_time:
+    description:
+    - The time to start the scheduler in format HH:MM:SS
+    - If no time is provided, midnight "00:00:00" will be used.
+    type: str
+  interval_unit:
+    description:
+    - The interval unit type
+    choices: [ hours, days ]
+    type: str
+  interval_length:
```

Comment:
UI uses frequency so I think we should use frequency_length

Reply:
ok